source/faq.txt

```diff
@@ -26,20 +26,25 @@ following the steps in :ref:`c2c-reconfigure-mid-migration`.
 Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
 ---------------------------------------------------------------------------------------
 
-You can perform reads at any time during synchronization. 
-However, ``mongosync`` combines and reorders writes from the source to destination during synchronization, 
-and also temporarily modifies various collection characteristics. 
-As a result, the destination is not guaranteed to match the source at any point in time when the sync is running. 
-For consistent reads, wait for the migration to :ref:`c2c-api-commit`. To learn more, see :ref:`mongosync-considerations`. 
-
-Performing writes to your destination cluster during synchronization results in undefined behavior. 
-``mongosync`` blocks writes on the destination cluster by default. To learn
-more about write-blocking, see :ref:`c2c-write-blocking` and :ref:`c2c-api-start`.
-
-Upon commit, it is only safe to write to the destination cluster when ``canWrite`` is ``true``. 
-To check the value of ``canWrite``, run the :ref:`c2c-api-progress` endpoint.
-
-To learn more about permissable reads and writes during synchronization, see :ref:`c2c-reads-and-writes`. 
+``mongosync`` combines and reorders writes from the source to destination during 
+synchronization, and temporarily modifies collection characteristics. As a 
+result, ``mongosync`` can't guarantee that the destination matches the source, 
+including a stale version of the source, at any point in time when the sync is 
+running. To safely accept traffic to the destination cluster, wait for the 
```

Comment:
"running" -> "running, even if the sync is paused"

Reply:
updated